packages/data-sdk/src/Fleet.ts

```diff
@@ -130,4 +133,8 @@ export class Fleet {
   static queryDevices = queryDevices;
   static queryEvents = queryEvents;
   static queryTelemetry = queryTelemetry;
+  static createDevice = createDevice;
+  static patchDevice = patchDevice;
+  static getDevicesData = getDevicesData;
+  static queryDevicesData = queryDevicesData;
```

Comment:
yea, these are interesting… `Fleet` is kinda a grab-bag of "functions" that don't always seems to relate to any _entity_—if you will—of **fleety-ness**. But I think this is "right", given the current architecture.

You _could_ also approach it as `static` on `Device`:

```ts
class Device {
  static createDevice = createDevice;
  static patchDevice = patchDevice;
  static getDevicesData = getDevicesData;
  static queryDevicesData = queryDevicesData;

  /* ...instance methods and whatnot... */
}
```

Then you _could do_:

```ts
Device.createDevice()
```

whether or not thats better than `Fleet.createDevice()` is… _questionable_.